hub-templates/base-hub/templates/docs-service-config.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nginx-docs-configmap
+  labels:
+    app: docs-service
+data:
+  nginx.conf: |
+    server {
+      listen 12222;
```

Comment:
I generally like to listen on port 8080, as the most popular [alternative http port](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=http-alt).